src/rgw/rgw_sal_motr.h

```diff
@@ -507,6 +507,20 @@ struct AccumulateIOCtxt{
   uint64_t total_bufer_sz = 0;
 };
 
+class MotrGetDataCB_Wrapper : public RGWGetDataCB {
+  RGWGetDataCB* cb;
```

Comment:
Hi, we don't need the `RGWGetDataCB` ptr, since we are inheriting from the same, we can call its functions directly.
eg. `RGWGetDataCB::handle_data()`

Reply:
"RGWGetDataCB" is abstract class.

Reply:
okay got it.. so we need a wrapper for any/all concrete class of `RGWGetDataCB` 👍🏼